slasher/cache/highest_attestation_cache_test.go

```diff
@@ -0,0 +1,108 @@
+package cache
+
+import (
+	"testing"
+
+	ethereum_slashing "github.com/prysmaticlabs/prysm/proto/slashing"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil/require"
+)
+
+func TestStoringAndFetching(t *testing.T) {
+	cache, err := NewHighestAttestationCache(10, nil)
+	require.NoError(t, err)
+
+	// cache
+	cache.Set(1, &ethereum_slashing.HighestAttestation{
+		ValidatorId:        1,
+		HighestSourceEpoch: 2,
+		HighestTargetEpoch: 3,
+	})
+
+	// has
```

Comment:
Full english sentences as comments are preferred. Adding comments as delimiters is probably not a good idea, comments should explain sth non-trivial. Such laconic comments serve little purpose explaining things (they are too short for that). Even in tests, let's remove obvious comments (for example `// delete` followed by delete operation serves no purpose).